### ISSUE TITLE
Opening POS won't proceed, stuck at loading.

### ISSUE THREAD

**Issue by Cyaaaaaaaaan** [nexopos/v3-issues#362]
**Username** : 
cyaaaaaaaaan

**Describe the bug**
Opening POS won't proceed, got stuck in loading.

**To Reproduce**
1. Set a default customer so I can proceed.
2. Clicked Open POS

**Expected behavior**
That it would just open the POS instead of getting stuck at loading

**Screenshots**
https://imgur.com/a/PMZLwsh

**Environment (please complete the following information):**
 - NexoPOS: 3.15.44
 - Gastro (if installed) [e.g. 2.3.10]
 - PHP version: 7.2.19-Win32-VC15-x64
 - MySQL version: 5.7.24-winx64
 - Devise used: Microsoft Edge Chromium-based (Desktop)

**Comment by nexopos** [comment]
Hi, It looks like your issue does'nt actually match the required format.
                Please note that you can choose between the following format : 
                    - [Issue](https://github.com/nexopos/v3-issues/issues/new?template=bug_report.md).
                    - [Feature Submition](https://github.com/nexopos/v3-issues/issues/new?template=feature_request.md).
                You might consider updating your issue and use the right version format. 
                Regards.

**Comment by Blair2004** [comment]
Hi, if your system is hosted online, please share the credentials to **contact@nexopos.com**. Don't forget to link this thread as well. If it's locally, we need to schedule remote assistance. We're available from 9am - 6pm GMT+1

Kind Regards.

**Comment by Blair2004** [comment]
By way, can you explain the steps you went through for the installation ?

**Comment by Cyaaaaaaaaan** [comment]
I hosted it locally.

- I used laragon as my environment.
- and then directed the root folder to the nexopos 3 folder.
- Added the database using the database button.
- Launched it through web button.
- Configured the database.
- Logged in to the app.

Everything's working fine except when opening the POS

**Comment by Blair2004** [comment]
Sorry for the late answer, does the issue has been solved so far ?

**Comment by Cyaaaaaaaaan** [comment]
Yes I did. I emailed the support on my new problem. Please consider looking
at it. Thank you!
